conf/profile.conf.js

```diff
@@ -217,6 +217,13 @@ exports.config = {
       userFieldSelector: '#j_username',
       passFieldSelector: '#j_password',
       logonButtonSelector: '#logOnFormSubmit'
+    },
+    'github-form': {
+      name: './authenticator/githubAuthenticator',
+      userFieldSelector: '#login_field',
+      passFieldSelector: '#password',
+      logonButtonSelector: 'input[type=submit]',
+      conditionalLogonButtonSelector: 'button[type=submit]'
```

Comment:
this selector has a different purpose, I'd suggest adding a new one e.g. authorizationButtonSelector